.gitlab/benchmarks.yml

```diff
@@ -8,14 +9,41 @@ include:
  file: 'images/templates/gitlab/notify-slo-breaches.template.yml'
 
 variables:
-  MACROBENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:cpp-nginx
+  MACROBENCHMARKS_CI_IMAGE: registry.ddbuild.io/images/benchmarking-platform-tools-ubuntu:latest
```

Comment:
This was correct, the image here should remain `MACROBENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:cpp-nginx`

Reply:
<img width="2000" height="1000" alt="image" src="https://github.com/user-attachments/assets/224ac30a-2716-4c73-8c23-86a0713e457a" />

Ok, I have no idea what's going on at this point

Reply:
```suggestion
  MACROBENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:cpp-nginx
```

Reply:
fixed in 0b672c6ca6c39d402503b490a39cf214d94a04e8